openstack/intel-worker/playbooks/roles/virtual_machine_create/templates/worker-amd64.ign.j2

```diff
@@ -0,0 +1,49 @@
+{
+	"ignition": {
+		"version": "3.4.0",
+		"config": {
+			"merge": [
+				{
+					"source": "http://{{ virtual_machine_create_bastion_ip }}:{{ virtual_machine_create_http_port }}/ignition/worker-amd64.ign"
```

Comment:
```suggestion
					"source": "http://{{ bastion_ip }}:{{ http_port }}/ignition/worker-amd64.ign"
```

Reply:
Ansible lint expects to have a role name as a prefix for the variable names .